### PR TITLE
set home:welcome:show key to false in local storage

### LINKED ISSUE #13: Add sample data “Sample eCommerce orders”

### PR DESCRIPTION
**Pull request by Tengda-He**
### Description
set home:welcome:show key to false in chrome local storage to achieve consistency in homepage display. Without this key the homepage display a welcome popup for first time visit.
 
### Issues Resolved
#13 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/opensearch-dashboards/without-security/dashboard_sanity_test_spec.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ describe('dashboard sample data validation', () => {
         before(() => {
             // Go to the home page
             miscUtils.visitPage('app/home#')
+            cy.window().then(win => win.localStorage.setItem('home:welcome:show', false))
```
Comment (tianleh) [inline cypress/integration/opensearch-dashboards/without-security/dashboard_sanity_test_spec.js:29]: Nice fix!

Can you also undo this change in the `after()` function? This is to ensure every test case is cleaning up its own data.

**Review by seraphjiang — APPROVED** [review]

File: cypress/integration/opensearch-dashboards/without-security/dashboard_sanity_test_spec.js (hunk 1 above)
Comment (Tengda-He) [reply cypress/integration/opensearch-dashboards/without-security/dashboard_sanity_test_spec.js:29]: add the remove in after section

**Review by tianleh — APPROVED** [review]
